README.md

```diff
@@ -112,3 +112,13 @@ Secrets which have been applied to the dogfooding cluster but are not committed
     to the image registry on GCP.
 - Lots of other secrets, hopefully we can add more documentation on them
   here as we go.
+
+#### Dogfooding Node Pools
```

Comment:
It might be worth expanding this a bit to explain why you might want one pool vs. the other. Should everything be using workload-id?

Reply:
Cheers, I've added a couple sentences of further explanation.